statsd/statsd.go

```diff
@@ -126,6 +126,9 @@ const (
 	writerWindowsPipe string = "pipe"
 )
 
+// noTimestamp is used as a value for metric without a given timestamp.
+const noTimestamp = int64(0)
```

Comment:
Good catch.
I think it may be simpler to change the `appendTimestamp` implementation to this test instead:
```
	if timestamp >= noTimestamp {
```
WDYT?

Reply:
Switched to documented guidelines: `timestamp` must be > 0 to be appended to a message or to be considered valid by `GaugeWithTimestamp` and `CountWithTimestamp`.